waiter/integration/waiter/token_request_test.clj

```diff
@@ -428,14 +428,6 @@
                   (assert-response-status response 412)
                   (is (str/includes? (str body) "Cannot modify stale token"))))
 
-              (testing "hard-delete without etag"
```

Comment:
Can we keep this test, or is there a change in behavior here?

Reply:
Added back the integration test.